packages/docs/render-cards.mjs

```diff
@@ -2,19 +2,130 @@ import { bundle } from "@remotion/bundler";
 import { getCompositions, renderStill } from "@remotion/renderer";
 import path from "path";
 
-const serveUrl = await bundle({
+// render cards
+const serveUrl_1 = await bundle({
   entryPoint: path.join(process.cwd(), "./src/remotion/entry.ts"),
   publicDir: path.join(process.cwd(), "static"),
 });
-const compositions = await getCompositions(serveUrl);
+const compositions_1 = await getCompositions(serveUrl_1);
 
-for (const composition of compositions.filter(
+for (const composition of compositions_1.filter(
   (c) => c.id.startsWith("expert") || c.id.startsWith("template")
 )) {
   await renderStill({
     composition,
     output: `static/generated/${composition.id}.png`,
-    serveUrl,
+    serveUrl: serveUrl_1,
   });
   console.log("Rendered", composition.id);
 }
+
+
+// extract articles
+import { execSync } from "child_process";
```

Comment:
It's a common convention to put all imports at the top, so it is immediately visible what dependencies this file has